cli/src/main/java/org/commonjava/maven/ext/cli/Cli.java

```diff
@@ -256,23 +256,26 @@ else if ( new File( target.getParentFile(), ManipulationManager.MARKER_FILE ).ex
             {
                 List<ArtifactRef> ts = RESTCollector.establishAllDependencies( session, pomIO.parseProject( session.getPom() ),
                                                                               profiles ).stream().sorted().collect(Collectors.toList());
-                logger.info( "Found {} dependencies", ts.size());
+                logger.info( "Found {} dependencies\n", ts.size());
+                System.out.format("\u001B[32m%-80s%-20s%-20s%-20s\033[0m%n",
+                                  StringUtils.center( "GAV", 60 ), "TYPE", "CLASSIFIER", "SCOPE" );
 
                 for ( ArtifactRef a : ts )
                 {
                     if ( a instanceof SimpleScopedArtifactRef )
                     {
-                        boolean isPlugin =  "maven-plugin".equals( a.getTypeAndClassifier().getType() );
-                        String scope = ( (SimpleScopedArtifactRef) a ).getScope() == null ? "compile" : ( (SimpleScopedArtifactRef) a ).getScope();
-                        System.out.format("%-80s%-20s%-20s%n", a.asProjectVersionRef(),
+                        final boolean isPlugin =  "maven-plugin".equals( a.getTypeAndClassifier().getType() );
+                        final String scope = ( (SimpleScopedArtifactRef) a ).getScope() == null ? "compile" : ( (SimpleScopedArtifactRef) a ).getScope();
+                        System.out.format("%-80s%-20s%-20s%-20s%n", a.asProjectVersionRef(),
                                           a.getTypeAndClassifier().getType(),
+                                          a.getTypeAndClassifier().getClassifier() == null ? "" : a.getTypeAndClassifier().getClassifier(),
                                           isPlugin ? "" : scope);
                     }
                 }
             }
             else if (printManipulatorOrder)
             {
-                System.out.println ("Manipulator order is:");
+                System.out.println ("\u001B[32mManipulator order is:\033[0m");
```

Comment:
I was thinking maybe we had Jansi to help with this, but it's only in test scope `org.fusesource.jansi:jansi:jar:1.13:test`.

Reply:
I'm guessing if we wait for picocli to go int hen we could use `org.fusesource.jansi.AnsiConsole`.

Reply:
It seems excessive to add jansi for two lines of code.